### ISSUE TITLE
Add correct 3D Maxwellian formulae to docs

### ISSUE THREAD

**Issue by dstansby** [PlasmaPy/PlasmaPy#392]
Currently the 3D Maxwellian docstrings have the formulae for 1D distributions; these should be replaced by the correct ones for 3D distributions:

http://plasmapy.readthedocs.io/en/master/api/plasmapy.physics.distribution.Maxwellian_velocity_3D.html#plasmapy.physics.distribution.Maxwellian_velocity_3D
http://plasmapy.readthedocs.io/en/master/api/plasmapy.physics.distribution.Maxwellian_speed_3D.html#plasmapy.physics.distribution.Maxwellian_speed_3D

**Comment by lemmatum** [comment]
This issue may need to be reformulated based on the discussion in Riot. I think the functions are written correctly, though the docstrings probably need clarification.

You mentioned wanting a 1 speed value argument for 3D. This might be achievable as a thin wrapper around the current 3D speed distribution function. You would take the one speed and then split it evenly among the 3 speed component arguments and call `Maxwellian_speed_3D` on it, I think. Though I don't think this kind of speed distribution will normalize correctly with a 1D integral (just integrating speed once), because the 3D normalization coefficient takes into account 3 integrations. The normalization issue is part of why I wrote `Maxwellian_speed_3D` the way I did, with 3 arguments.
